lib/tasks/errbit/database.rake

```diff
@@ -10,6 +10,12 @@ namespace :errbit do
     puts "=== Cleared #{ResolvedProblemClearer.new.execute} resolved errors from the database."
   end
 
+  desc "Delete old errors from the database. (Useful for limited heroku databases)"
+  task :clear_outdated => :environment do
+    require 'outdated_problem_clearer'
+    puts "=== Cleared #{OutdatedProblemClearer.new.execute} outdated errors from the database."
```

Comment:
This seems good. It might be nice to include some more informational output in a switch here to help guide people to this feature if they're looking for it. For example:
```ruby
if Errbit.config.errbit_problem_destroy_after_days.present?
  puts "=== Cleared #{OutdatedProblemClearer.new.execute} outdated errors from the database."
else
  puts "=== ERRBIT_PROBLEM_DESTROY_AFTER_DAYS not set. Old problems will not be destroyed."
end